util/admin/admin.go

```diff
@@ -104,6 +104,9 @@ func CancelJobs(txn kv.Transaction, ids []int64) ([]error, error) {
 				errs[i] = errors.Trace(err)
 				continue
 			}
+			if job.Type == model.ActionAddIndex {
+				t.SetJobListKey(meta.AddIndexJobListKey)
+			}
```

Comment:
Add an else branch so that the code is more robust without the assumption about `job.Type` default value.